go.mod

```diff
@@ -4,8 +4,10 @@ go 1.21.6
 
 require (
 	github.com/simonvetter/modbus v1.6.0
-	go.viam.com/rdk v0.20.1-0.20240209215422-1764cb9007e8
-	go.viam.com/utils v0.1.61
+	github.com/stretchr/testify v1.9.0
+	go.viam.com/api v0.1.302
+	go.viam.com/rdk v0.27.1-0.20240517182344-8789c0b8d6a9
```

Comment:
v0.27.1 was released last week. If we get rid of the hyphen and everything after it, does this PR still work? If so, we don't need to wait until next week's release...

Reply:
yep, we need to wait then will i will bump this to v0.28 and merge

Reply:
the analog read changes are in next weeks release